examples/with-apollo/lib/with-apollo.js

```diff
@@ -27,19 +27,21 @@ function withApollo (PageComponent, { ssr = true } = {}) {
 
   WithApollo.displayName = `withApollo(${getDisplayName(PageComponent)})`
 
-  if (ssr) {
-    WithApollo.getInitialProps = async ctx => {
-      const { AppTree } = ctx
+  // Allow Next.js to remove getInitialProps from the browser build
+  if (typeof window === 'undefined') {
```

Comment:
Hi there! Pardon my ignorance while I ask this question.

Why was `WithApollo.getInitialProps` removed from the client? 
Unless I set Apollo up wrong (which I very well could have as I'm _extremely_ new to Apollo and pretty new to Next and SSR in general), this kills any `getInitialProps` call on any PageComponent wrapped with this HOC when using client-side navigation. Is that the purpose? Seems strange to pretty much remove that feature right?

Reply:
Great catch! 🙏 #8620 8620